src/web/routes/application/children-dob/children-dob.js

```diff
@@ -12,13 +16,39 @@ const pageContent = ({ translate }) => ({
   aboutYourChild: translate('childrenDob.aboutYourChild')
 })
 
+const behaviourForGet = (req, res, next) => {
+  if (!req.session.hasOwnProperty('children')) {
+    req.session.children = {}
+  }
+
+  req.session.children.count = 1
+
+  res.locals.children = req.session.children
+  next()
+}
+
+const extractChildrenEntries = (val, key) => key.startsWith('children')
+
+const behaviourForPost = (req, res, next) => {
+  if (req.body.hasOwnProperty('add')) {
```

Comment:
It feels like we should have this in a function so its clearer what its doing, something like `addActionRequested`, but that's not quite right.

Reply:
Agreed - I would like to add the actions to a map:
```
{
  add: () => {},
  remove: () => {}
}
```
.. but not sure that will work for this situation